app/main/browser.js

```diff
@@ -55,7 +55,13 @@ let currentState;
 
 function sendToAllWindows(event: string, args: Object): void {
   console.log(`browser.js: sendToAllWindows ${JSON.stringify(args)}`);
-  store.getState().browserWindows.forEach((id) => {
+  const windows = store.getState().browserWindows;
```

Comment:
Hmm.  I'd prefer to guarantee that the state always includes an empty list of `browserWindows`, although initial states in Redux seem to be trickier than anticipated.  This is fine for now.